### ISSUE TITLE
function name typo

### ISSUE THREAD

**Issue by MuuZh** [benfulcher/TimeSeriesGeneration#2]
https://github.com/benfulcher/TimeSeriesGeneration/blob/71e393fb1f3596e8a74f9ed35ddf1468a328ab2c/HCTSA_generation/createDataAndHCTSAMat.m#L280

TS_init is not recognized in my compiler (MATLAB R2021b on Windows 10)

Since it's defined as TS_Init in HCTSA, maybe use the original name is a batter option in case sensitive scenario?
> _https://github.com/benfulcher/hctsa/blob/eada66a0e9b46dae51abae8b887e5777bec7f56e/Calculation/TS_Init.m#L1_

**Comment by benfulcher** [comment]
Yes, it should be capitalized. Do you know how to send a pull request to fix? Else I'll edit

**Comment by MuuZh** [comment]
I'll have a look of other files and see if there's any other minor issue and summarize in a single pull :)
